src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -1498,12 +1500,150 @@ bool emberAfColorControlClusterStopMoveStepCallback(EndpointId aEndpoint, app::C
     return true;
 }
 
-bool emberAfColorControlClusterColorLoopSetCallback(EndpointId endpoint, app::CommandHandler * commandObj, uint8_t updateFlags,
-                                                    uint8_t action, uint8_t direction, uint16_t time, uint16_t startHue,
-                                                    uint8_t optionsMask, uint8_t optionsOverride)
+static void startColorLoop(EndpointId endpoint, uint8_t startFromStartHue)
 {
-    // TODO IMPLEMENT LOGIC
-    emberAfSendImmediateDefaultResponse(EMBER_ZCL_STATUS_UNSUP_COMMAND);
+    uint8_t direction = 0;
+    Attributes::GetColorLoopDirection(endpoint, &direction);
+
+    uint16_t time = 0x0019;
+    Attributes::GetColorLoopTime(endpoint, &time);
```

Comment:
As a followup, it sure would be nice if we had the accessors do defaults themselves instead of forcing those to be duplicated in manually written code.  Please file that as an issue?

Reply:
Created the issue for accessor functions.
#8994